components/scroller/scroller.stories.js

```diff
@@ -0,0 +1,102 @@
+import DtScroller from './scroller';
+import BaseScrollerMdx from './scroller.mdx';
+import ScrollerDefault from './scroller_default.story';
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+export const argTypesData = {
```

Comment:
add

```js
import { action } from '@storybook/addon-actions';

export const argsData = {
  onScrollStart: action('scroll-start'),
  onScrollEnd: action('scroll-end'),
};
```

pass it into the default export object with `args: argsData,`

Reply:
Mhmm I added it a clean it a little it but I cannot trigger the events on the `action` tab in storybook.. it can be because the event its emmited from `core-scroller` component and not on `scroller`?

Reply:
see https://github.com/dialpad/dialtone-vue/pull/780#discussion_r1134544113 I think that will fix your issue here.

Reply:
Still not work, we can follow the discussion there and resolve this conflict to void multithread

https://github.com/dialpad/dialtone-vue/pull/780#discussion_r1134701244